pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -1616,4 +1616,33 @@ public void testGetTxnState() throws Exception {
         Transaction abortingTxn = transaction;
         Awaitility.await().until(() -> abortingTxn.getState() == Transaction.State.ABORTING);
     }
+    @Test
+    public void testDeleteTopicPolicyWhenDeleteSystemTopic() throws Exception {
```

Comment:
please move this test to [NamespacesTest.java](https://github.com/apache/pulsar/pull/18823/files#diff-24788d54292c43e74fe60920b6edd12c0526b6cf5808aec9ac5a134c554db31a)